module/User/src/Module.php

```diff
@@ -58,18 +52,6 @@ public function onBootstrap($event)
                 return $response;
             }
         }, -100);
-
-        $eventManager->attach(MvcEvent::EVENT_DISPATCH_ERROR, function ($e) use ($authService) {
-            // Check if not a console route, as the console has no headers.
-            if (!($e->getResponse() instanceof Response)) {
-                if (!$authService->hasIdentity()) {
-                    $response = $e->getResponse();
-                    $response->getHeaders()->addHeaderLine('Location', '/user');
-                    $response->setStatusCode(302);
-                    return $response;
-                }
-            }
-        });
```

Comment:
I do think that we can and perhaps should keep this, but without checking `$e->getResponse()` of course.

Reply:
That's also fine!